runtime/doc/news.txt

```diff
@@ -42,6 +42,9 @@ DIAGNOSTICS
 
 • |vim.diagnostic.config()| accepts a "jump" table to specify defaults for
   |vim.diagnostic.jump()|.
+• Diagnostics are filtered by severity before being passed to a diagnostic
```

Comment:
I think we should also document this in [`diagnostic.txt`](https://github.com/neovim/neovim/blob/9e0d40f7e45f483e54d38be1266f63240808b4b0/runtime/doc/diagnostic.txt#L73).

Reply:
Good call, added.